packages/stylelint-config/package.json

```diff
@@ -9,17 +9,12 @@
   ],
   "license": "Apache-2.0",
   "dependencies": {
-    "@stylelint/postcss-css-in-js": "^0.37.2",
     "postcss": "^8.3.11",
-    "postcss-syntax": "^0.36.2",
-    "prettier": "2.4.1",
-    "stylelint": "14.0.1",
+    "postcss-styled-syntax": "^0.6.4",
+    "stylelint": "^15.0.0",
     "stylelint-config-clean-order": "^5.0.0",
-    "stylelint-config-prettier": "^9.0.3",
-    "stylelint-config-styled-components": "^0.1.1",
-    "stylelint-order": "^6.0.3",
-    "stylelint-prettier": "^1.2.0",
-    "stylelint-processor-styled-components": "^1.10.0"
+    "stylelint-config-standard": "^34.0.0",
+    "stylelint-order": "^6.0.3"
```

Comment:
Previous styled-components integrations is now deprecated so migrated to `postcss-styled-syntax`